### PR TITLE
Metaproteomics Workflow Testing

### PR DESCRIPTION
**Pull request by jraysajulga**
Hello! Just getting the conversation started with our workflows for testing. Not sure if this is the way to go about it, but here we go.

Also, I am not sure how to test the yaml file, so I might need some guidance with this one. For example, I am not sure if the labels for the inputs/outputs are correct. I can add "assert" tests once I am confident with the labeling. 

The output history for this workflow is [here](http://129.114.16.192/u/pjagtap/h/metaproteomics-training-output-1). You will need to log in/register to view it.

### CODE REVIEW

File: training/proteomics/metaproteomics/metaproteomics.ga [hunk 1]
```diff
@@ -0,0 +1,836 @@
...
+                "owner": "iuc", 
+                "tool_shed": "toolshed.g2.bx.psu.edu"
+            }, 
+            "tool_state": "{\"sqlquery\": \"\\\"SELECT go.description, \\\\ncount(distinct bering_psms.sequence) as \\\\\\\"bering_peptides\\\\\\\", count(distinct bering_psms.id) as \\\\\\\"bering_psms\\\\\\\" \\\\nFROM go JOIN bering_prot_go ON go.go_id = bering_prot_go.go_reference JOIN bering_prot on bering_prot_go.id = bering_prot.id JOIN bering_psms ON bering_prot.peptide = bering_psms.sequence\\\\nWHERE go.aspect = 'biological_process'\\\\nGROUP BY go.description\\\\nORDER BY  bering_peptides desc,bering_psms desc\\\"\", \"__page__\": null, \"sqlitedb\": \"{\\\"__class__\\\": \\\"RuntimeValue\\\"}\", \"query_result\": \"{\\\"header\\\": \\\"yes\\\", \\\"header_prefix\\\": \\\"35\\\", \\\"__current_case__\\\": 0}\", \"__rerun_remap_job_id__\": null}", 
+            "tool_version": null, 
+            "type": "tool", 
+            "uuid": "9d9634a3-436e-44fc-b43d-a606d1feae7b", 
+            "workflow_outputs": [
+                {
+                    "label": null,
```
Comment (bgruening) [inline training/proteomics/metaproteomics/metaproteomics.ga]: the label here needs to be `query_results` as you referring to it in https://github.com/usegalaxy-eu/workflow-testing/pull/34/files#diff-4e0d654ce7c7570ef760bca55ae37f52R26

**Comment by jraysajulga** [comment]
Thanks @bgruening. That's really helpful. When I tested it, I got this error message:

`Failed to find output [None] in invocation outputs [{u'go_terms_cellular_localization': {u'src': u'hda', u'id': u'2c6b71ef30ff00fd'}}]`

**Comment by jraysajulga** [comment]
@bgruening I installed planemo-0.58.0.dev0 and tested again... Same issue.

**Comment by jraysajulga** [comment]
@bgruening I might have fixed it. I removed the unused "workflow_outputs" information. I will keep testing to make sure.

**Comment by jraysajulga** [comment]
@bgruening Looks like it works now. I added more tests as well.

**Comment by bgruening** [comment]
Cool thanks. I will let you know if we get it running automatically.

**Comment by bgruening** [comment]
Cool, see the new badge on the front page :) Thanks a lot, keep them coming!

**Comment by jraysajulga** [comment]
Wonderful! Looks great. 

Thank you. Will do.

**Pull request merged**